docs/src/data/links.ts

```diff
@@ -105,62 +120,112 @@ export const inputComponents = [
     href: '/components/textfield',
     label: 'Text Field',
     body: `The TextField form primitive can be used allow users to input text content.`,
+    platforms: ['react'],
```

Comment:
I can't comment on the line, but TextAreaField was a new primitive added, you'll need to add the platforms array to it. It shows up when Flutter is active on the docs site.